src/sidekick/module.js

```diff
@@ -1119,6 +1119,7 @@
       sk.isEditor() ? '' : sk.location.pathname,
     );
     loginUrl.searchParams.set('loginRedirect', 'https://www.hlx.live/tools/sidekick/login-success');
+    loginUrl.searchParams.set('extensionId', chrome.runtime.id);
```

Comment:
I think we'll need to keep suppporting the 3rd party cookie way of authenticating for the bookmarklets:
```suggestion
    const extensionId = chrome && chrome.runtime && chrome.runtime.id;
    if (extensionId) {
      loginUrl.searchParams.set('extensionId', extensionId);
    }
```

Reply:
right.